src/main/protobuf-generated/cpp/include/genomicsdb_vid_mapping.pb.h

```diff
@@ -598,46 +599,46 @@ inline void InfoField::clear_has_length() {
   _has_bits_[0] &= ~0x00000008u;
 }
 inline void InfoField::clear_length() {
-  length_.ClearToEmptyNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited());
+  length_.ClearToDefaultNoArena(_default_length_);
   clear_has_length();
 }
 inline const ::std::string& InfoField::length() const {
   // @@protoc_insertion_point(field_get:InfoField.length)
-  return length_.GetNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited());
+  return length_.GetNoArena(_default_length_);
 }
 inline void InfoField::set_length(const ::std::string& value) {
   set_has_length();
-  length_.SetNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited(), value);
+  length_.SetNoArena(_default_length_, value);
   // @@protoc_insertion_point(field_set:InfoField.length)
 }
 inline void InfoField::set_length(const char* value) {
   set_has_length();
-  length_.SetNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited(), ::std::string(value));
+  length_.SetNoArena(_default_length_, ::std::string(value));
   // @@protoc_insertion_point(field_set_char:InfoField.length)
 }
 inline void InfoField::set_length(const char* value, size_t size) {
   set_has_length();
-  length_.SetNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited(),
+  length_.SetNoArena(_default_length_,
       ::std::string(reinterpret_cast<const char*>(value), size));
   // @@protoc_insertion_point(field_set_pointer:InfoField.length)
 }
 inline ::std::string* InfoField::mutable_length() {
   set_has_length();
   // @@protoc_insertion_point(field_mutable:InfoField.length)
-  return length_.MutableNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited());
+  return length_.MutableNoArena(_default_length_);
 }
 inline ::std::string* InfoField::release_length() {
   // @@protoc_insertion_point(field_release:InfoField.length)
   clear_has_length();
-  return length_.ReleaseNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited());
+  return length_.ReleaseNoArena(_default_length_);
 }
 inline void InfoField::set_allocated_length(::std::string* length) {
   if (length != NULL) {
     set_has_length();
   } else {
     clear_has_length();
   }
-  length_.SetAllocatedNoArena(&::google::protobuf::internal::GetEmptyStringAlreadyInited(), length);
+  length_.SetAllocatedNoArena(_default_length_, length);
   // @@protoc_insertion_point(field_set_allocated:InfoField.length)
 }
 
```

Comment:
Instead of editing a generated file, let's make changes in the proto message definitions.